t.sentinel.import/t.sentinel.import.py

```diff
@@ -496,8 +533,41 @@ def main():
             cloudlist.append(vect)
             grass.run_command('g.copy', vector=vect + '@' + new_mapset + ',' + vect)
         for rast in grass.parse_command('g.list', type='raster', mapset=new_mapset):
-            maplist.append(rast)
-            grass.run_command('g.copy', raster=rast + '@' + new_mapset + ',' + rast)
+            grass.run_command('g.copy',
+                              raster=rast + '@' + new_mapset + ',' + rast)
+            if "CLOUDS" in rast:
+                cloudlist.append(rast)
+            else:
+                maplist.append(rast)
+
+                if options["offset"]:
+                    # save the description.json
+                    tmp_desc_dir = os.path.join(tmpdirectory, "descriptions_json")
+                    if not os.path.isdir(tmp_desc_dir):
+                        try:
+                            os.makedirs(tmp_desc_dir)
+                        except:
+                            grass.fatal(_(f"Unable to create directory {tmp_desc_dir}"))
+
+                    desc_file_save = os.path.join(tmp_desc_dir, f"{rast}_description.json")
+                    desc_file_in = os.path.join(json_standard_folder, rast, "description.json")
+                    shutil.copy(desc_file_in, desc_file_save)
```

Comment:
not sure if needed but with `shutil.copy2` metadata, like the file’s creation and modification times, is preserved (see [here](https://docs.python.org/3/library/shutil.html#shutil.copy2)).

Reply:
Thanks, I think I will leave it as it is though because it might be even better to make GRASS think the `description.json` is a new file, instead of an old an modified one ;)